images2gif.py

```diff
@@ -0,0 +1,1095 @@
+# -*- coding: utf-8 -*-
+#   Copyright (C) 2012, Almar Klein, Ant1, Marius van Voorden
```

Comment:
It would be nice to know the source of this code so that it can be updated if a new version is released.